src/dos/cdrom_image.cpp

```diff
@@ -188,6 +289,17 @@ bool CDROM_Interface_Image::GetAudioTracks(int& stTrack, int& end, TMSF& leadOut
 	stTrack = 1;
 	end = (int)(tracks.size() - 1);
 	frames_to_msf(tracks[tracks.size() - 1].start + 150, &leadOut.min, &leadOut.sec, &leadOut.fr);
+
+#ifdef DEBUG
+	LOG_MSG("%s CDROM: GetAudioTracks, stTrack=%d, end=%d, leadOut.min=%d, leadOut.sec=%d, leadOut.fr=%d",
+      get_time(),
+	  stTrack,
+	  end,
+	  leadOut.min,
+	  leadOut.sec,
+	  leadOut.fr);
+#endif
```

Comment:
Format string is long enough to be split into 2 lines, I think.

Parameters would look better if indented with 2 tabs instead of a tab + spaces.

Reply:
Big cleanup in b6dc0476; everything is aligned properly now regardless of tab width.